app/provisioning_client.go

```diff
@@ -0,0 +1,128 @@
+package app
+
+import (
+	"context"
+
+	pb "go.viam.com/api/provisioning/v1"
+	"go.viam.com/utils/rpc"
+)
+
+// ProvisioningInfo holds provisioning info.
+type ProvisioningInfo struct {
+	FragmentID   string
+	Model        string
+	Manufacturer string
+}
+
+func provisioningInfoFromProto(info *pb.ProvisioningInfo) *ProvisioningInfo {
+	return &ProvisioningInfo{
+		FragmentID:   info.FragmentId,
+		Model:        info.Model,
+		Manufacturer: info.Manufacturer,
+	}
+}
+
+// NetworkInfo holds network information.
+type NetworkInfo struct {
+	Type      string
+	SSID      string
+	Security  string
+	Signal    int32
```

Comment:
I think having an int32 here is a little annoying for users, there's slight friction of having to always do `int32({SomeValue})` instead of just `SomeValue`. Can we make this just an int and do the conversion for users under the hood?

Reply:
Oh! Thanks for catching that. That was before we started switching everything to ints, so I'm glad to get this done.